action/action-simulator/src/main/java/com/powsybl/action/simulator/loadflow/LoadFlowActionSimulatorLogPrinter.java

```diff
@@ -61,25 +62,37 @@ public void loadFlowConverged(RunningContext runningContext, List<LimitViolation
             out.println(Security.printLimitsViolations(violations, runningContext.getNetwork(), LoadFlowActionSimulator.NO_FILTER));
         }
     }
+    private static void addKeyValueToTable(AbstractTableFormatter formatter, String key, String value) {
+        try {
+            formatter.writeCell(key);
+            formatter.writeCell(value);
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+
+    }
 
     @Override
     public void ruleChecked(RunningContext runningContext, Rule rule, RuleEvaluationStatus status, Map<String, Object> variables, Map<String, Boolean> actions) {
         if (verbose || status == RuleEvaluationStatus.TRUE) {
             out.println("        Rule '" + rule.getId() + "' evaluated to " + status);
         }
         if (verbose &&  (variables.size() + actions.size() > 0)) {
-            Table table = new Table(2, BorderStyle.CLASSIC_WIDE);
-            table.addCell("Variable");
-            table.addCell("Value");
-            variables.forEach((key, value) -> {
-                table.addCell(key);
-                table.addCell(Objects.toString(value));
-            });
-            actions.forEach((key, value) -> {
-                table.addCell(key + ".actionTaken");
-                table.addCell(value.toString());
-            });
-            out.println(table.render());
+
+            try (Writer myWriter = new StringWriter();
+                 AbstractTableFormatter formatter = new AsciiTableFormatter(myWriter, "myFormatter", 2)) {
+                formatter.writeCell("Variable");
+                formatter.writeCell("Value");
+                variables.forEach((key, value) ->
+                        addKeyValueToTable(formatter, key + "", value.toString())
+                );
+                actions.forEach((key, value) ->
+                        addKeyValueToTable(formatter, key + ".actionTaken", value.toString())
+                );
+                out.println(myWriter.toString());
```

Comment:
Could you instanciate the TableFormatter from the `out` instead of using a temporary StringWriter (using a PrintWriter for instance)